### PR TITLE
[draft] Deployment Options

### PR DESCRIPTION
**Pull request by LadyBluenotes**
Related to issue #331

### CODE REVIEW

**Comment by LadyBluenotes** [comment]
@jccpalmer

File: content/guides/deploying-your-app.mdx [hunk 1]
```diff
@@ -0,0 +1,384 @@
+# Deployment Options
+
+Are you ready to deploy your Solid application? Follow our guides to different deployment services.
```
Comment (jccpalmer) [inline content/guides/deploying-your-app.mdx:3]:
```suggestion
Are you ready to deploy your Solid application? Follow our guides for different deployment services.
```

Comment (jccpalmer) [reply content/guides/deploying-your-app.mdx:3]: While not wrong, "guides to" doesn't sound quite right to me. "Guides for" sounds and flows better, in my opinion. Feel free to disregard if you wish.

File: content/guides/deploying-your-app.mdx [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+
+
+## Cloudflare
+
+[Cloudflare Pages](https://pages.cloudflare.com/) is a JAMstack platform for frontend developers, where JAMstack stands for JavaScript, APIs, and Markup. 
+For additional details and features, you can [visit the Cloudflare website](https://pages.cloudflare.com/).
+
+### Using the Cloudflare's web interface
+
+1. Navigate to the [Cloudflare log up page](https://dash.cloudflare.com/login) and log in or sign up.
```
Comment (jccpalmer) [inline content/guides/deploying-your-app.mdx:91]:
```suggestion
1. Navigate to the [Cloudflare login page](https://dash.cloudflare.com/login) and log in or sign up.
```

Comment (jccpalmer) [reply content/guides/deploying-your-app.mdx:91]: Yes?

File: content/guides/deploying-your-app.mdx [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+[](https://app.eraser.io/workspace/w9y9PNVjwSqDCEPNTEoe?elements=XcbVyX2a69kSAP1m1220Ug)
+
+3.  You'll have the option to install Cloudflare Pages on all your repositories or select ones. 
+Choose the repository that contains your Solid project.
+
+[](https://app.eraser.io/workspace/w9y9PNVjwSqDCEPNTEoe?elements=SsbGUghc_Vwlvxefe1xAFg)
+
+4. Configure your build settings:
+- The project name will default to the repository name, but you can change it if you wish.
+- The "build command" field, enter `npm run build` .
```
Comment (jccpalmer) [inline content/guides/deploying-your-app.mdx:107]:
```suggestion
- In the "build command" field, enter `npm run build` .
```

File: content/guides/deploying-your-app.mdx [hunk 4]
```diff
@@ -0,0 +1,384 @@
...
+Make sure to navigate to the `Speed` -> `Optimization settings` section in your Cloudflare website dashboard and disable the `Auto Minify` option. 
+This is important as minification and comment removal can interfere with hydration.
+
+## Firebase
+
+[Firebase](https://firebase.google.com/) is an all-in-one app development platform by Google, offering a range of services from real-time databases to user authentication. 
+For a detailed overview of the services available, you can visit [Firebase's documentation](https://firebase.google.com/docs).
+
+Before proceeding, make sure you've already set up a project in your Firebase console. 
+If haven't, you can follow [Firebase's official guide](https://firebase.google.com/docs/projects/learn-more#creating-cloud-projects) to create a new Firebase project.
```
Comment (jccpalmer) [inline content/guides/deploying-your-app.mdx:160]:
```suggestion
If you haven't, you can follow [Firebase's official guide](https://firebase.google.com/docs/projects/learn-more#creating-cloud-projects) to create a new Firebase project.
```

File: content/guides/deploying-your-app.mdx [hunk 5]
```diff
@@ -0,0 +1,384 @@
...
+
+```bash
+npm i -g firebase-tools
+# or
+pnpm i -g firebase-tools
+# or
+yarn global add firebase-tools
+```
+
+2. Execute the `firebase login` command to ensure you're logged into the Firebase account associated with your project.
```
Comment (jccpalmer) [inline content/guides/deploying-your-app.mdx:174]:
```suggestion
2. Execute the `firebase login` command to ensure that you're logged into the Firebase account associated with your project.
```

File: content/guides/deploying-your-app.mdx [hunk 6]
```diff
@@ -0,0 +1,384 @@
...
+```
+
+**Note:** 
+Before proceeding, ensure that your Netlify account and team are fully set up. 
+This is crucial for a seamless project setup and deployment.
+
+2. Open your terminal, navigate to your project directory, and run the `netlify init` command. 
+Authenticate using one of the supported login options.
+
+3.  Follow the on-screen instructions from the CLI. When prompted for the 'Directory to deploy,' specify `dist` — this is where Solid stores the built project files.
```
Comment (jccpalmer) [inline content/guides/deploying-your-app.mdx:247]:
```suggestion
3. Follow the on-screen instructions from the CLI. When prompted for the 'Directory to deploy,' specify `dist` — this is where Solid stores the built project files.
```

File: content/guides/deploying-your-app.mdx [hunk 7]
```diff
@@ -0,0 +1,384 @@
...
+yarn global add @railway/cli
+```
+
+2. Open your terminal and run the following command to log in:
+
+```bash
+railway login
+```
+
+3. You have the option to link your local Solid project to an existing Railway project using railway link.
```
Comment (jccpalmer) [inline content/guides/deploying-your-app.mdx:322]: > railway link

Is this a proper noun? Should it be capitalized?

File: content/guides/deploying-your-app.mdx [hunk 8]
```diff
@@ -0,0 +1,384 @@
...
+[](https://app.eraser.io/workspace/w9y9PNVjwSqDCEPNTEoe?elements=0mwBl275l6WC3YD5Uz_IcQ)
+
+2. Once on the dashboard, click the button at the top right corner and choose "Add New Project." 
+On the next page, select "Continue with GitHub" or your preferred Git service.
+
+[](https://app.eraser.io/workspace/w9y9PNVjwSqDCEPNTEoe?elements=fafx0siiy-6dTa5dpDlhbw)
+
+3. You will then see with a list of your repositories.
+Use the search bar if needed to find the specific repository you want to deploy. 
+Click the import button to proceed.
```
Comment (jccpalmer) [inline content/guides/deploying-your-app.mdx:353]:
```suggestion
Click the "Import" button to proceed.
```

Comment (jccpalmer) [reply content/guides/deploying-your-app.mdx:353]: I assume that's what the button says.

**Review by jccpalmer — COMMENTED** [review]
Edits.

File: content/guides/deploying-your-app.mdx (hunk 7 above)
Comment (LadyBluenotes) [reply content/guides/deploying-your-app.mdx:322]: it's a cli command so no capitalization

**Pull request merged**